app/src/main/res/values-ro/strings.xml

```diff
@@ -0,0 +1,49 @@
+<!--
+  ~ Copyright 2019 Google LLC
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<resources>
+    <string name="menu_filter_by_grow_zone">Filtrează pe zona de creștere</string>
+    <string name="plant_detail_image_content_description">Imaginea plantei</string>
+    <string name="my_garden_title">Grădina mea</string>
+    <string name="plant_list_title">Lista plantelor</string>
+    <string name="available">Plante disponibile</string>
+    <string name="plant_details_title">Detaliile plantei</string>
+    <string name="add_plant">Adăuga plantă</string>
+    <string name="added_plant_to_garden">Plantă adăugată în grădină</string>
+    <string name="garden_empty">Grădina ta este goală</string>
+    <string name="plant_date_header">Plantată</string>
+    <string name="watered_date_header">Ultima oară udată</string>
+    <string name="menu_item_share_plant">Împarte</string>
+    <string name="share_text_plant">Schau dir die %s Pflanze in der App "Sunflower" an</string>
```

Comment:
This text looks suspiciously similar to German ;)

Reply:
Unfortunately I don't speak german to know that similarity :) but probably that is because some words derives from latin

Reply:
Madalin, nic0lette pointed out that you used the german translation for the _share_text_plant_ string. 
A rough translation would be:
_Vezi planta %s în aplicația Android Sunflower_

Reply:
I'm completely blind, you are right :), thanks